examples/multiphysics/rayleigh_benard_convection/rayleigh_benard_convection.prm

```diff
@@ -0,0 +1,166 @@
+subsection initial conditions
+    set type 				= nodal
+    subsection uvwp
+            set Function expression 	= 0; 0; 0
+    end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  set number of fluids 			= 1
+  subsection fluid 0
+  	set density 			= 100
+  	set kinematic viscosity 	= 0.01
+  	set thermal expansion 		= 0.0002
+  	set thermal conductivity 	= 0.15625	# for Ra = 10000
+  	#set thermal conductivity 	= 0.0625	# for Ra = 25000
+  	set specific heat  		= 100
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+	set type 		= dealii
+	set grid type 		= subdivided_hyper_rectangle
+	set grid arguments 	= 2, 1: 0.5, 0.25 : 0 , 0 : true
+	set initial refinement 	= 6
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+subsection multiphysics
+	set buoyancy force = true
+	set heat transfer  = true
+	set fluid dynamics = true
+end
+
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 4
+    subsection bc 0
+    set id = 0
+        set type              = noslip
+    end
+    subsection bc 1
+    set id = 1
+        set type              = noslip
+    end
+    subsection bc 2
+    set id = 2
+        set type              = noslip
+    end
+    subsection bc 3
+    set id = 3
+        set type              = noslip
+    end
+end
+
+subsection boundary conditions heat transfer
+  set number                  = 4
+    subsection bc 0
+    set id = 0
+        set type	      = convection-radiation
+        set h	              = 0
+        set Tinf	      = 0
+        set emissivity        = 0
+    end
+    subsection bc 1
+    set id = 1
+        set type	      = convection-radiation
+        set h	              = 0
+        set Tinf	      = 0
+        set emissivity        = 0
+    end
+        subsection bc 2
+    set id = 2
+        set type              = temperature
+	set value             = 50
+    end
+    subsection bc 3
+    set id = 3
+        set type              = temperature
+	set value             = 0
+    end
+end
+
+# --------------------------------------------------
+# Source term
+#---------------------------------------------------
+subsection source term
+  set enable                 		= true
+  subsection xyz
+            set Function expression 	= 0 ; -10 ; 0
+  end
+end
+
+# --------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+subsection non-linear solver
+  set verbosity               = verbose
+  set tolerance               = 1e-8
+  set max iterations          = 20
```

Comment:
Are that many needed?

Reply:
At some steps where we the vortices are formed the solver needs more iterations (more than 10). Other than that, it needs 4-5 iterations.